server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftScanClientHandler.java

```diff
@@ -253,7 +279,9 @@ private ScanResult continueScan(TInfo tinfo, long scanID, SingleScanSession scan
     } catch (CancellationException ce) {
       server.getSessionManager().removeSession(scanID);
       Tablet tablet = server.getOnlineTablet(scanSession.extent);
```

Comment:
Ran into an exception while testing.  Think the following is a fix for it.

```suggestion
      TabletBase tablet = scanSession.getTabletResolver().getTablet(scanSession.extent);
```

Reply:
Fix applied in ec7cbad